src/legacy/ui/public/vislib/lib/dispatch.js

```diff
@@ -112,23 +139,22 @@ export function VislibLibDispatchProvider(Private, config) {
       const handler = this.handler;
       const color = get(handler, 'data.color');
       const config =  handler && handler.visConfig;
-      const valueAxes = config && config.get('valueAxes');
-      const isPercentageMode = valueAxes && valueAxes.some(({ scale }) => scale && scale.mode === 'percentage');
+      const isPercentageMode = this.isSeriesInPercentageMode(d.seriesId, series, config);
```

Comment:
will this work with pies, heatmaps and gauges ?

Reply:
heatmap doesn't like it

> Error: Accessing invalid config property: seriesParams (http://localhost:5603/zcv/bundles/commons.bundle.js:172399)

VisConfig is quite strict about accessing unset properties, throwing an Exception

so, there's a `get` and a `set` method . how about adding a `has` method? 

Of course making get a bit more liberal would also be an option, maybe even adding a default value param? :). 

However, safer would be the `has` option

Reply:
Oh, there's a default param in the `VisConfig` `get` method 👍